### PR TITLE
Mades updates to official typings so removed TODO's and bumped @types/googlemaps version number

### PR DESCRIPTION
**Pull request by Rennzie**
# Please explain PR reason.

A small update to package numbers and removing @ ts-ignore from the TO DO's relating to missing official typings. These typings have been merged and published to npm

### CODE REVIEW

**Comment by JustFly1984** [comment]
Please don’t change package version, it will be set at publication time

**Comment by Rennzie** [comment]
> Please don’t change package version, it will be set at publication time

sorry about that. Uriklar had said to do it when I chatted with hime a few days back.

**Comment by uriklar** [comment]
Hi @Rennzie, I meant the version of ```@types/googlemaps```. Sorry for the misunderstanding

**Comment by Rennzie** [comment]
> Hi @Rennzie, I meant the version of `@types/googlemaps`. Sorry for the misunderstanding

Ah my bad, I did that also haha. Ok cool, think I've got it now.

File: packages/react-google-maps-api/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-google-maps/api",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (JustFly1984) [inline packages/react-google-maps-api/package.json]: please revert back

**Pull request merged**